### PR TITLE
Remove mascot footer image 

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

This PR removes all settings related to the mascot footer image. Note that after removing these settings there were only 2 left in `Settings::Mascot`. so it no longer makes sense to have a dedicated model for that so I moved the remaining settings back to `SiteConfig`. Note that we can only remove the `settings_mascots` table after this PR has been deployed across the fleet.

## Related Tickets & Documents

#13587

## QA Instructions, Screenshots, Recordings

Go to http://localhost:3000/admin/customization/config and ensure the Mascot section works as expected.

### UI accessibility concerns?

None

## Added tests?

- [X] Yes, updated existing ones
 
## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [X] I'm not sure how best to communicate this change and need help

## [optional] Are there any post deployment tasks we need to perform?

- [ ] Ping @citizen428 after this got deployed across the fleet so we can remove the settings table

### CODE REVIEW

File: lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class MoveMascotSettingsBacktoSiteConfig
+    def run
+      return unless Database.table_available?("settings_mascots")
+
+      # Remove obsolete data update script
```
Comment (juliannatetreault) [inline lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb:6]: Yay, for comments!

**Review by juliannatetreault — APPROVED** [review]
Works like a charm! ✨

File: lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      DataUpdateScript.delete_by(file_name: "20210420050256_move_mascot_settings")
+
+      # Migrate back explicitly set values
+      mascot_image_url = SiteConfig.mascot_image_url
```
Comment (mstruve) [inline lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb:10]: I think this needs to be `Settings::Mascot.image_url` and below needs to be `Settings::Mascot.mascot_user_id`

**Review by mstruve — COMMENTED** [review]
I think you replaced one too many `Settings::Mascot` in the data update script ;)

File: lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb (hunk 2 above)
Comment (citizen428) [reply lib/data_update_scripts/20210504060704_move_mascot_settings_backto_site_config.rb:10]: Ugh, that's very likely, a lot of the search/replace was done with `sed` and I must have missed this one when doing a manual review at the end. Thanks!

File: app/models/site_config.rb [hunk 3]
```diff
@@ -103,10 +103,6 @@ class SiteConfig < RailsSettings::Base
         type: :string,
         default: proc { URL.local_image("mascot.png") },
         validates: { url: true }
-  field :mascot_image_description, type: :string, default: "The community mascot"
-  field :mascot_footer_image_url, type: :string, validates: { url: true }
-  field :mascot_footer_image_width, type: :integer, default: 52
-  field :mascot_footer_image_height, type: :integer, default: 120
```
Comment (rhymes) [inline app/models/site_config.rb:109]: should we delete these explicitly with the data update script?

**Review by rhymes — APPROVED** [review]

File: app/models/site_config.rb (hunk 3 above)
Comment (citizen428) [reply app/models/site_config.rb:109]: I'll do all of that in the follow-up PR where I also remove the `settings_mascot` table.

**Pull request merged**